src/Tools/AI Test Toolkit/src/AITCommandLineCard.Page.al

```diff
@@ -0,0 +1,103 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.AITestToolkit;
+
+using System.Environment;
+
+page 149042 "AIT CommandLine Card"
```

Comment:
We have namespaces for a reason, lets stop the prefixing of objects.
```suggestion
page 149042 "CommandLine Card"
```

Reply:
@darjoo But doesn't AL still check the object name only? [AL0197](https://learn.microsoft.com/de-de/dynamics365/business-central/dev-itpro/developer/diagnostics/diagnostic-al197)

Reply:
Yes and no.
As this would be in a completely different namespace, that's perfectly fine. 
If you include this as a dependency and your app isn't namespaced, then yes, it will clash.

Reply:
@darjoo But doesn't it mean that you may shorten names only once namespaces are mandatory for everybody? You cannot just create a breaking scenario for many, at least not without any announcement.